### PR TITLE
Add Research page Changes

### LINKED ISSUE #218: Add Research Page

### PR DESCRIPTION
**Pull request by RitwikSrivastava**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #218 

Test URLs:
- Original: https://www.sunstar.com/research/
- Before: https://main--sunstar--hlxsites.hlx.page/_drafts/ritsriva/research1
- After: https://researchpage--sunstar--hlxsites.hlx.page/_drafts/ritsriva/research1

- [ ]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://<branch>--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/<your-block>&index=0

### CODE REVIEW

**Comment by JiangLong2019** [comment]
Some comments.

1. Looks like the cards block is broken.
image

2. Scientific Literature content shows different.
image

3. Text overlap image area lose title and shows different from original site.
image

4. Lose R&D Stories content
image

5. A scroll bar shows in mobile view.
image

**Comment by RitwikSrivastava** [comment]
> Some comments.

> 2. Scientific Literature content shows different.
I made it consistent with the already available block . and icon not appearing issue i have fixed. 

> 3. Text overlap image area lose title and shows different from original site.
The text is now visible and iam keeping it consitent with the already available hero block instead of creating a variant as discussed.

> 4. Lose R&D Stories content
This needs to be worked upon. It will be a feed of Stories. 

> 5. A scroll bar shows in mobile view.
Fixed. 
@JiangLong2019

File: blocks/carousel/carousel.css [hunk 1]
```diff
@@ -56,6 +76,24 @@
...
+  letter-spacing: 0;
+  font-weight: var(--font-weight-medium);
+  color: var(--black);
+}
+
+.carousel p {
+  margin-bottom: 1.875rem;
+}
+
+.carousel.small p{
```
Comment (sdmcraft) [inline blocks/carousel/carousel.css:93]: Looks like incorrect indentation

File: blocks/carousel/carousel.css [hunk 2]
```diff
@@ -56,6 +76,24 @@
...
+  font-size: 14px;
```
Comment (sdmcraft) [inline blocks/carousel/carousel.css:94]: Use global variables for font-size from styles.css

File: blocks/columns/columns.css [hunk 3]
```diff
@@ -451,3 +457,13 @@
     flex-direction: row;
   }
 }
+
+@media (max-width: 61rem){
```
Comment (sdmcraft) [inline blocks/columns/columns.css:461]: Please use mobile first `mid-width` media queries.

**Comment by sdmcraft** [comment]
Some comments from UI review

Screenshot 2023-10-27 at 10 29 17 AM
Screenshot 2023-10-27 at 10 26 58 AM
Screenshot 2023-10-27 at 10 26 00 AM
Screenshot 2023-10-27 at 10 24 33 AM
Screenshot 2023-10-27 at 10 23 56 AM
Screenshot 2023-10-27 at 10 21 30 AM
Screenshot 2023-10-27 at 10 19 03 AM

**Comment by RitwikSrivastava** [comment]
@sdmcraft Please re-review. All changes made (margins, image size in mobile views) and also the mobile first media queries.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**